internal/services/housekeeper/sourcesCleaner.go

```diff
@@ -32,6 +34,7 @@ func StartSourcesListenerService() {
 }
 
 func sourcesListener(msg *k.Message, _ *k.Consumer) {
+	db := database.GetDB()
 	headers := msg.Headers
 	for _, v := range headers {
 		if v.Key == "event_type" && string(v.Value) == "Application.destroy" {
```

Comment:
The "Application.destroy" happens after successfully deleting the app and not just triggering the destroy/delete action?

Reply:
Not sure if I get you question but `Application.destroy` event is triggered when cluster is removed from the sources in consoledot.